README.md

```diff
@@ -167,18 +168,20 @@ Before you begin, make sure you have the following installed:
  Step 1: 
   Git Clone the Repository 
   
-    git clone https://github.com/adi-kiran/burnout-healthgoalsapp.git
+    git clone https://github.com/aditikilledar/burnout-proj3
 
 Step 2:
    Ensure mongodb is running and execute the below to populate the tables:
-    
+   Navigate to the backend folder using "cd backend".
+
+  Once in the backend, run:
+  
     python insert_event_data.py
```

Comment:
Can we have the "cd backend" command before "python insert_event_data.py" in the commands section?

Reply:
sure, it's actually already in the sentence above, but I'll add it again.

Reply:
Maybe we can remove it from the previous line. Just so it is part of the command set that users can directly copy to clipboard and run from the documentation. 
Changes can be implemented in the new branch.